### PR TITLE
fix(api): save user email on the first try

### LINKED ISSUE #227: new user add email addr need save twice
### Description

import emby account,and edit there email,when first time click save button,lost input email and back to user name,but after input the email addr again,the save button works

### Version

1.1.1

### Steps to Reproduce

As Description

### Screenshots

_No response_

### Logs

_No response_

### Platform

desktop

### Device

desktop pc

### Operating System

win11

### Browser

chrome

### Additional Context

_No response_

### Code of Conduct

- [X] I agree to follow Overseerr's Code of Conduct

### PR DESCRIPTION
**Pull request by gauthier-th**
#### Description

Fix email field in profile settings where it requires you to enter the email once, press save, and then to enter it again and re-save.

### To-Dos

- [x] Successful build `yarn build`
- [x] Translation keys `yarn i18n:extract`
- [ ] Database migration (if required)

#### Issues Fixed or Closed

- Fixes #227

### CODE REVIEW

File: server/routes/user/usersettings.ts [hunk 1]
```diff
@@ -127,20 +128,21 @@ userSettingsRoutes.post<
       user.settings.originalLanguage = req.body.originalLanguage;
       user.settings.watchlistSyncMovies = req.body.watchlistSyncMovies;
       user.settings.watchlistSyncTv = req.body.watchlistSyncTv;
-      user.email = req.body.email ?? user.email;
     }
 
-    await userRepository.save(user);
+    const savedUser = await userRepository.save(user);
+
+    // TODO: check if the email of the user has been modified after save (if the email already exists in the db)
```
Comment (fallenbagel) [inline server/routes/user/usersettings.ts]: Remove this todo

**Review by Fallenbagel — CHANGES_REQUESTED** [review]

**Review by Gauvino — APPROVED** [review]
Tested successfully, trying multiple account different manipulation everything I could and now work like a charm thanks you!

**Comment by Fallenbagel** [comment]
@all-contributors please add @gauthier-th  for code

**Pull request merged**